src/test/java/io/managed/services/test/rhoas/KafkaRhoasBasicTests.java

```diff
@@ -62,12 +63,14 @@ public class KafkaRhoasBasicTests extends TestBase {
 
     // TODO enterprise: change logic of name assignment back once we have quota for more than 1 enterprise kafka instance
     private static final String KAFKA_INSTANCE_NAME = Environment.IS_ENTERPRISE ? "enterprise-test" : "e2e-cli-test-" + Environment.LAUNCH_SUFFIX;
+    private static final String KAFKA_CREATE_INSTANCE_NAME = Environment.IS_ENTERPRISE ? "enterprise-test-kfk" : "e2e-test-kfk-" + Environment.LAUNCH_SUFFIX;
```

Comment:
change name to something more descriptive of fact that this kafka is created using cli & its only purpose is to try to change its owner.